metrics/ignores/platform-all.json

```diff
@@ -16,6 +16,7 @@
     "expression-tests/legacy/interval/composite-default": "https://github.com/mapbox/mapbox-gl-native/issues/12747",
     "expression-tests/legacy/interval/tokens-zoom": "https://github.com/mapbox/mapbox-gl-native/issues/12747",
     "expression-tests/resolved-locale/basic": "Even the 'en' locale may not be present on some test systems.",
+    "expression-tests/within/non-supported": "Something is wrong with GeoJSON geometry in test",
```

Comment:
It throws exception while loading test information (even before we actually run it): 
```
2023-05-05 21:47:43.550 mbgl-expression-test[20437:12390485] [ERROR] {}[General]: Cannot load test data from '/Users/siarheifedartsou/work/maplibre-gl-native/metrics/integration/expression-tests/within/non-supported/test.json': Polygon must be described by 4 or more coordinate points. Improper nesting can also lead to this error. Double check that the coordinates are properly nested and there are 4 or more coordinates.
```